scripts/cli/cmd/erc20.js

```diff
@@ -44,13 +73,17 @@ const transferCmd = new Command("transfer")
         await erc20Instance.approve(args.erc20HandlerAddress, args.value);
         console.log(`[ERC20 Transfer] Approved ${args.erc20HandlerAddress} to spend ${args.value} tokens from ${args.wallet.address}!`);
 
+        // Compute resourceID
+        resourceID = await erc20HandlerInstance._tokenContractAddressToResourceID(args.erc20Address)
+
         const data = '0x' +
-            ethers.utils.hexZeroPad(erc20Instance.address, 32).substr(2) +              // OriginHandlerAddress  (32 bytes)
+            resourceID.substr(2) +              // OriginHandlerAddress  (32 bytes)
```

Comment:
I think we really need to be able to specify the resourceId as an arg, otherwise this limits us to ethereum generated ones

Reply:
this retrieves it from what is stored in the current resourceID -> tokenAddress whitelist on chain, i don't think it limits us/